staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -3051,6 +3051,13 @@ type PodStatus struct {
 	// More info: https://git.k8s.io/community/contributors/design-proposals/node/resource-qos.md
 	// +optional
 	QOSClass PodQOSClass `json:"qosClass,omitempty" protobuf:"bytes,9,rep,name=qosClass"`
+	// NominatedNodeName is set when this pod preempts other pods on the node, but it cannot be
```

Comment:
These comments are translated to user-facing documentation. Either don't mention the field name or use the json field name (nominatedNodeName).

Reply:
Details to specify somewhere:
1. Will this field always be set before nodeName is set?
2. Will this field be unset when nodeName is set?
3. Will this field be changed if nodeName is set to something different?

Reply:
Fixed the comment and moved the field up. Also added one more line to the comment to mention that NominatedNodeName and NodeName are not necessarily the same. More details about how this field is set is provided in [preemption design doc](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/scheduling/pod-preemption.md#starvation-problem).